src/hyp_map.rs

```diff
@@ -323,6 +325,63 @@ impl UmodeInputRegion {
     }
 }
 
+/// Represents a hypervisor stack region.
+pub struct HypStackRegion {
+    vaddr: PageAddr<SupervisorVirt>,
+    paddr: PageAddr<SupervisorPhys>,
+    page_count: u64,
```

Comment:
once all stacks are the same size, we should be able to avoid storing vaddr & page_count for each CPU rigth?